aya-log/src/lib.rs

```diff
@@ -370,6 +413,12 @@ pub enum Error {
 
     #[error("invalid /sys/devices/system/cpu/online format")]
     InvalidOnlineCpu(#[source] io::Error),
+
+    #[error("program not found")]
+    ProgramNotFound,
+
+    #[error("error getting in use program maps")]
+    GetProgramMapIds(#[source] ProgramError),
```

Comment:
I think that this variant should be called ProgramError, and should be error(transparent)

Reply:
Reworked this change also